build/commands/lib/config.js

```diff
@@ -532,10 +532,14 @@ Config.prototype.buildArgs = function () {
     generate_about_credits: true,
   }
 
-  if (this.isOfficialBuild()) {
+  if (this.isOfficialBuild() || getEnvConfig(['enable_updater'])) {
     args.enable_updater = true
   }
 
+  if (getEnvConfig(['use_prebuilt_omaha4']) !== undefined) {
+    args.use_prebuilt_omaha4 = getEnvConfig(['use_prebuilt_omaha4'])
+  }
```

Comment:
I think you can just do:

```ts
args.use_prebuilt_omaha4 = getEnvConfig(['use_prebuilt_omaha4'])
```

Because anything set as `undefined` gets thrown out.

Reply:
yeah, this should just be a single line how @cdesouza-chromium wrote.

Reply:
just to be clear the same thing does not apply to `args.enable_updater = true` because the use of `this.isOfficialBuild()` means that the return value will not be undefined and we only want to set the value here if it doesn't match the default in gn